lms/services/vitalsource/service.py

```diff
@@ -183,15 +184,28 @@ def get_user_reference(self, lti_params: LTIParams) -> str | None:
 
         return value
 
-    def has_h_license(self, lti_user: LTIUser, lti_params: LTIParams) -> bool:
+    def check_h_license(
```

Comment:
Returning error codes here now.

It's probably not the best design but allows us to hide all the ugly VS details here and keep just one call in the LTI view.